test/e2e/curated_packages_test.go

```diff
@@ -273,3 +273,51 @@ func TestCPackagesVSphereKubernetes122BottleRocketSimpleFlow(t *testing.T) {
 	)
 	runCuratedPackageInstallSimpleFlow(test)
 }
+
+func TestCPackagesNutanixKubernetes120SimpleFlow(t *testing.T) {
+	test := framework.NewClusterE2ETest(t,
+		framework.NewNutanix(t),
+		framework.WithClusterFiller(api.WithKubernetesVersion(v1alpha1.Kube120)),
+		framework.WithPackageConfig(t, packageBundleURI(v1alpha1.Kube120),
+			eksaPackageControllerHelmChartName, eksaPackageControllerHelmURI,
+			eksaPackageControllerHelmVersion, eksaPackageControllerHelmValues),
+		framework.WithEnvVar("NUTANIX_PROVIDER", "true"),
```

Comment:
I could be way out of my league here, but would it make more sense for the `framework.NewNutanix(t)` call to set this for you?

Regardless, you might want to switch from "NUTANIX_PROVIDER" TO "features.NutanixProviderEnvVar" to prevent gotchas should that value ever change (however unlikely).